libs/community/langchain_community/document_loaders/parsers/audio.py

```diff
@@ -12,6 +12,144 @@
 logger = logging.getLogger(__name__)
 
 
+class AzureOpenAIWhisperParser(BaseBlobParser):
+    """Transcribe and parse audio files.
+
+    Audio transcription is with Azure OpenAI Whisper model.
+    This is different to the Open AI Whisper parser and requires
+    an Azure OpenAI API Key.
+
+    Args:
```

Comment:
I'm not sure if this is aligned to the docstring formatting used elsewhere - though I don't think this formatting is sphinx-compatible... but then maybe they're not using sphinx. So, if this is the formatting used for other integrations this that's all good.

Reply:
I've written up a better docstring I think covers most of what the class does and I've tried to format it similar to other azure services, let me know though if you think anything needs to be added/removed :)